tests/test_fake.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+import sys
+
+sys.path.append("local")
+import awkward_kaitai
+
+
+def test_fake():
+    animal = awkward_kaitai.Reader(
+        "test_artifacts/libfake.so"
+    )
+    awkward_array = animal.load("example_data/data/fake.raw")
+
+    assert len(awkward_array.fakeA__Zpoints.pointA__Zx[0]) == 3000
+    assert awkward_array.fakeA__Zpoints.pointA__Zy[0][0] == 2
+    assert awkward_array.fakeA__Zpoints.pointA__Zz.to_numpy().sum() == 66000
```

Comment:
@ManasviGoyal I just invented some simple tests.
Do you have some detailed docs on how you generated the files? or should I just continue doing tests like these also for other files?

Reply:
Are you referring to the raw data files? I think it would be better for atleast the simpler test files to check against the entire output like the test for animal.ksy. You can just change the raw data file names and the ksy file in the `README.md` instructions.

Reply:
string-based tests are a bit obscure,
in case of errors it is a pain to understand where the difference is.
Do you have the code that was used to generate the raw data files?

Reply:
Every test raw data file was generated differently. I have the code for fake.ksy but can't find the data for others.
```python
open("fake.raw", "wb").write(
    np.array(
        [(1, 2, 3), (10, 20, 30), (11, 22, 33)] * 1000,
        dtype=[("x", "u4"), ("y", "u4"), ("z", "u4")]
    ).tobytes()
)
```

If you want to see how the data should look like, then you can use [Kaitai Web IDE](https://ide.kaitai.io/)

Reply:
ok, I plan to transform to `numpy` and compare arrays